Gemfile

```diff
@@ -53,4 +53,4 @@ gem 'shoulda-matchers', group: [:development, :test], require: false
 gem 'devise'
 gem 'foundation-rails', '~> 6.5'
 gem 'webpacker', '~> 3.3'
-gem "active_model_serializers"
\ No newline at end of file
+gem "active_model_serializers"  
```

Comment:
when `add`ing files to be committed, I recommend taking a close look at the changes for each file. Here the gemfile was modified where trailing whitespace was added to the end of the line, which is not a change we want or need